src/structures/AnonymousGuild.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const BaseGuild = require('./BaseGuild');
+const { VerificationLevels, NSFWLevels } = require('../util/Constants');
+
+/**
+ * Bundles common attributes between
```

Comment:
Between my love for you?
```suggestion
 * Bundles common attributes between guild classes
```